vars/waitForContainer.groovy

```diff
@@ -0,0 +1,19 @@
+def call(String CONTAINER_NAME, String LOGS) {
+    sh """
+        set -o errexit
+        set -o xtrace
+        export CONTAINER_NAME=${CONTAINER_NAME}
+        export LOGS="${LOGS}"
+        attempt=0
+        while [ \$attempt -le 30 ]; do
+            attempt=\$(( \$attempt + 1 ))
+            echo "Waiting for ${CONTAINER_NAME} to be up (attempt: \$attempt)..."
+            result=\$(docker logs ${CONTAINER_NAME} 2>&1)
+            if grep -q "${LOGS}" <<< \$result ; then
+              echo "${CONTAINER_NAME} is ready!"
+              break
+            fi
+            sleep 1
+        done;
```

Comment:
Do we really need this loop to be executed case 30 times and printed 30 times message?
If we know that running containers took up to 30 seconds why not just sleep for 30 seconds and then check logs
Or at least lets make interval for 10 seconds. So the loop will be executed only 3 times.

Reply:
good point, I will change it to run with 10 seconds wait, I think it was done initially with 30 seconds of sleep, but Illia suggested improving it with container startup wait

Reply:
Done